### PR TITLE
Create mentor account

### PR DESCRIPTION
**Pull request by perlajarillo**
This PR includes the following:

- Mentor's role, which includes: account creation, authentication and headers.
- Setting url in pictureName attribute.
- Setting mentorState.
- Organization of components and operations related to mentors.
- Moving log in out of home page.
- Update of mentors admin view.

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const styles = theme => ({
 });
 
 const NavAuthUser = props => {
-  const { classes, mobileOpen, toggle, admin } = props;
+  const { classes, mobileOpen, toggle, rol } = props;
```
Comment (Fa-v) [inline src/components/Header/Header.js:53]: *typo `rol` should be `role`

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -97,6 +97,29 @@ const NavAuthUser = props => {
...
+              ) : rol === "mentor" ? (
+                <div>
+                  <List>
+                    <Button
+                      tabIndex="-1"
+                      color="inherit"
+                      component={Link}
+                      to="/mentorshome"
+                    >
+                      Set state
```
Comment (Fa-v) [inline src/components/Header/Header.js]: What about "Add status" or "Update status" sounds less "React-minded" 😜

**Review by Fa-v — APPROVED** [review]
Great job! I thinks the changes in the home page make easier to identify the mentor and the mentees log process.
Filter in the mentor page is working fine.

File: src/components/Header/Header.js (hunk 2 above)
Comment (perlajarillo) [reply src/components/Header/Header.js]: That is very true. I have changed it. 👍

**Pull request merged**